src/site/content/en/blog/codelab-sign-up-form-best-practices/index.md

```diff
@@ -0,0 +1,269 @@
+
+## Step 1: Help password managers securely suggest and store passwords
+
+In this step you'll learn how to use form elements and attributes to make the most of built-in 
+browser features.
+
+{% Instruction 'remix' %}
+
+* Add the following code to your `index.html` file inside the `<main>` element:
+
+```html
+<form action="#" method="post">
+  
+  <h1>Sign up</h1>
+  
+  <section>
+    <label>Full name</label>
+    <input>
+  </section>
+  
+  <section>
+    <label>Email</label>
+    <input>
+  </section>
+  
+  <section>
+    <label>Password</label>
+    <input>
+  </section>
+  
+  <button id="sign-up">Sign up</button>
+  
+</form>
+```
+
+Here's how your project should look at this point:
+
+{% Glitch {
+  id: 'sign-in-form-codelab-1',
+  path: 'index.html',
+  height: 220
+} %}
+
+Click **View App** to preview your sign-in form. 
+
+This shows you what a form looks like with no CSS other than the 
+[default browser styles](https://bitsofco.de/a-look-at-css-resets-in-2018). 
+
+{% Aside %}
+[Browser Default Styles](https://browserdefaultstyles.com) lists default CSS for HTML elements.
+{% endAside %}
+
+* Do the default styles *look* OK? What would you change to make the form look better?
+* Do the default styles *work* OK? What problems might be encountered using your form as it is? What 
+about on mobile? What about for screenreaders or other [assistive technologies](/a11y-tips-for-web-dev)?
+* Who are your users, and what devices and browsers are you targeting? 
+
+### How will you test your form? 
+
+You could acquire a lot of hardware and set up a 
+[device lab](https://www.smashingmagazine.com/2016/11/worlds-best-open-device-labs/), but there are 
+cheaper and simpler ways to try out your form on a range of browsers and platforms:
+
+* [Use Chrome DevTools Device Mode](https://developers.google.com/web/tools/chrome-devtools/device-mode) 
+to simulate mobile devices. 
+* [Send the URL from your computer to your phone](https://support.google.com/chrome/answer/9430554).
+* Use a service such as [BrowserStack](https://www.browserstack.com/open-source) to test on a range 
+of devices and browsers. 
+
+{% Aside 'caution' %}
+Testing your site on a range of devices and browsers is especially important for forms, because 
+[small problems can cause high bounce rates](https://baymard.com/checkout-usability) and cause users 
+to give up on creating an account or completing a purchase.
+{% endAside %}
+
+Click **View Source** to return to your source code.
+
+## Step 2: Add CSS to make the form work better
+
+There's nothing wrong with the HTML you added, but you need to make sure your form works well for a 
+range of users on mobile and desktop.
+
+In this step you'll add CSS to make the form easier to use.
+
+Copy and paste all the following CSS into your own `style.css` file:
+
+{% Glitch {
+  id: 'sign-up-form-codelab-2',
+  path: 'css/main.css',
+  height: 400,
+  previewSize: 0
+} %}
```

Comment:
I wish this was a regular code block, the embedded Glitch tends to take a while to load. Just an observation, I understand you chose this to limit the height and we don't have that kind of custom css for code blocks.

Reply:
I wanted to use a code block, too, but it's a lot of code :(. 

I've now boosted this Glitch, which might improve things a bit.